src/config/dark-sites.config

```diff
@@ -501,13 +511,16 @@ vrv.co
 vuecinemas.nl
 w0rp.com
 w41k3r.com
+wallhaven.cc
+wallpaperplay.com
 wannabuy.biz
 warcraftdaily.com
 warezforums.com
 warsow.net
 wasm.continuation-labs.com/d3demo/
 weavesilk.com
 weboas.is
+websitecarbon.com
```

Comment:
It's just bright on my eyes, unsure for you but It flashed in my opinion the brightness of thee colors.

Reply:
Doesn't happen to me. But I'll remove it.